src/main/java/com/google/cloud/teleport/templates/PubsubToText.java

```diff
@@ -74,11 +113,24 @@
    * <p>Inherits standard configuration options.</p>
    */
   public interface Options extends PipelineOptions, StreamingOptions {
+    @Description(
+        "The Cloud Pub/Sub subscription to consume from. "
+            + "The name should be in the format of "
+            + "projects/<project-id>/subscriptions/<subscription-name>.")
+    ValueProvider<String> getInputSubscription();
+
+    void setInputSubscription(ValueProvider<String> value);
+
     @Description("The Cloud Pub/Sub topic to read from.")
-    @Required
     ValueProvider<String> getInputTopic();
     void setInputTopic(ValueProvider<String> value);
 
+    @Description(
+        "This determines whether the template reads from "
+        + "a pub/sub subscription or a topic")
+    @Default.Boolean(false)
+    Boolean getUseSubscription();
+
```

Comment:
@n-oden 

 Can you include setter for useSubscription option ``` void setUseSubscription(Boolean value);```

 Rebase to latest master version and verify all tests are executed successfully using ```mvn test```

Reply:
Oops, done.  test output:

```
[WARNING] Tests run: 255, Failures: 0, Errors: 0, Skipped: 7
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:39 min
[INFO] Finished at: 2021-05-03T10:05:31-04:00
[INFO] ------------------------------------------------------------------------
```